src/utils/entrypoints.js

```diff
@@ -16,10 +17,19 @@ module.exports = (src) => {
   }
 
   return fs.readdirSync(pathToEntryPoints).reduce(
-    (accumulator, file) => ({
-      ...accumulator,
-      [file.split('.')[0]]: path.resolve(pathToEntryPoints, file),
-    }),
-    {},
-  );
+    (accumulator, file) => {
+      // If types are provided, only watch/build those.
+      if (filteredEntrypoints.length > 0) {
+        const type = file.split('.')[0];
```

Comment:
Because you're doing the same thing twice, here and on L32, you can probably move this outside of the `if()` statement on L22 and reuse `type` for L32.